cmake/checks/check_02_compiler_features.cmake

```diff
@@ -501,3 +501,44 @@ IF(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
 
   RESET_CMAKE_REQUIRED()
 ENDIF()
+
+
+#
+# CCache support is only really useful for developers. 
+# We therefore disable the feature by default.
+#
+# See https://invent.kde.org/kde/konsole/-/merge_requests/26/diffs
+# and https://stackoverflow.com/a/24305849
+#
+SET(BUILD_USING_CCACHE FALSE CACHE BOOL "Enable ccache support")
```

Comment:
All other cached variables are recorded in `setup_cached_variables.cmake`. So it should be refactored in there.